bridge-processor/src/main/java/org/springframework/cloud/stream/module/bridge/BridgeProcessor.java

```diff
@@ -22,12 +22,12 @@
 import org.springframework.messaging.Message;
 
 /**
- * A Processor module that returns messages that is passed in for performance testing.
+ * A Processor application that connects input and output channels.
  *
  * @author Glenn Renfro
  */
 @EnableBinding(Processor.class)
-public class NoOpProcessor {
+public class BridgeProcessor {
 
 	@Transformer(inputChannel = Processor.INPUT, outputChannel = Processor.OUTPUT)
 	public Object transform(Message<?> message) {
```

Comment:
I may be a bit late in this game, but usage of `BridgeHandler` would really be better. As for this processor premise as well as by the performance avoiding SpEL method evaluations.